content/how-to/_index.md

```diff
@@ -1,7 +1,7 @@
 ---
 title: "User guide"
-description: ""
-lead: ""
+description: 
```

Comment:
I like to preserve the empty quotes so that if somebody does decide to add a description, they're reminded to quote it.